src/main/java/org/spin/form/import_file_loader/ImportFileLoaderServiceLogic.java

```diff
@@ -439,27 +444,39 @@ public static ListEntitiesResponse.Builder listFilePreview(ListFilePreviewReques
 			throw new AdempiereException("@FileImportNoFormat@");
 		}
 
-		// validate attachment reference
-		int attachmentReferenceId = request.getResourceId();
-		if (attachmentReferenceId <= 0) {
-			throw new AdempiereException("@FillMandatory@ @AD_AttachmentReference_ID@");
-		}
+		// Get File Name
+		String fileName = request.getResourceName();
 
-		byte[] file = AttachmentUtil.getInstance()
-			.withClientId(Env.getAD_Client_ID(Env.getCtx()))
-			.withAttachmentReferenceId(attachmentReferenceId)
-			.getAttachment();
-		if (file == null) {
-			throw new AdempiereException("@NotFound@");
+		if (fileName == null) {
+			throw new AdempiereException("@FileResourceName");
 		}
 
+
 		String charsetValue = request.getCharset();
 		if (Util.isEmpty(charsetValue, true) || !Charset.isSupported(charsetValue)) {
 			charsetValue = Charset.defaultCharset().name();
 		}
 		Charset charset = Charset.forName(charsetValue);
+		MClientInfo clientInfo = MClientInfo.get(Env.getCtx());
+		// Connector S3
+		MADAppRegistration genericConnector = MADAppRegistration.getById(
+			Env.getCtx(),
+			clientInfo.getFileHandler_ID(),
+			null
+		);
```

Comment:
please validate `genericConector` and launch exception with 
```java
if (genericConnector == null || genericConnector.getAD_AppRegistration_ID() <= 0) {
    throw new AdempiereException("@AD_AppSupport_ID@ @NotFound@");
}
```

Reply:
add validations https://github.com/solop-develop/adempiere-grpc-server/pull/856/commits/3f3b29d589036557e935a0e1a4a25a05dd8d15b4